tests/E2E Tests/WebAppUiTests/B2CWebAppCallsWebApiLocally.cs

```diff
@@ -77,8 +70,19 @@ public async Task Susi_B2C_LocalAccount_TodoAppFucntionsCorrectly()
             IBrowserContext context = await browser.NewContextAsync(new BrowserNewContextOptions { IgnoreHTTPSErrors = true });
             await context.Tracing.StartAsync(new() { Screenshots = true, Snapshots = true, Sources = true });
 
+            Process? serviceProcess= null;
+            Process? clientProcess = null;
+
             try
             {
+                // Start the web app and api processes.
+                // The delay before starting client prevents transient devbox issue where the client fails to load the first time after rebuilding.
+                // The delay after processes are started gives time to finish initial setup before attempted connection.
+                serviceProcess = UiTestHelpers.StartProcessLocally(_testAssemblyPath, _devAppPath + TC.s_todoListServicePath, TC.s_todoListServiceExe, serviceEnvVars);
+                Thread.Sleep(3000);
+                clientProcess = UiTestHelpers.StartProcessLocally(_testAssemblyPath, _devAppPath + TC.s_todoListClientPath, TC.s_todoListClientExe, clientEnvVars);
+                Thread.Sleep(5000);
```

Comment:
The second wait was required as a result of the changes because Playwright was erroring out when the connection was refused as the apps were still starting up (afaik Playwright won't keep retrying in the case of refused connections, it retries for interactions with actual sites). However, I have toyed with the idea of putting it in a try catch loop to retry at a faster cadence to right-size the wait so I'll add that and see what you think.

Reply:
Doing the try-catch loop in local testing resulted in only 1 to 2 seconds spent waiting.

Reply:
I still see the second "wait 5 seconds" in the code.

Reply:
Thanks, I missed taking out that line in one of the tests, removed now.